pages/desktop/consumer_pages/details.py

```diff
@@ -40,15 +40,15 @@ class Details(Base):
     _report_abuse_button_locator = (By.CSS_SELECTOR, '.button.abuse')
     _report_abuse_box_locator = (By.CSS_SELECTOR, '.abuse-form')
 
-    def __init__(self, testsetup, app_name=False, first_access=False):
+    def __init__(self, testsetup, app_name=None):
         Base.__init__(self, testsetup)
-        if first_access:
```

Comment:
No, it doesn't make a lot of sense to me, especially as the wait is not an element to not be present/displayed.